WcaOnRails/config/locales/en.yml

```diff
@@ -1265,17 +1265,17 @@ en:
       name_html: "WCA Board"
       info: "(WCA organization, delegate reports, competition approval, general inquiries)"
     wct:
-      info: "(communications of the WCA with the community and the general public)"
+      info: "The WCT is in charge of overseeing and supporting communications of the WCA with the community and general public. They should be contacted if you have general questions about the WCA or their social media appearance."
     wdc:
-      info: "(independent advice to WCA board, independent investigations, counselor for important personal matters)"
+      info: "The WDC investigates a variety of situations and proposes solutions, including punishments when necessary.  Among other topics, these situations could be serious violations of WCA regulations or community issues. If you have experienced any of these, feel free to approach this committee."
     wrc:
-      info: "(maintenance of the WCA Regulations)"
+      info: "The WRC handles all issues which are related to the application, the improvement and the development of the WCA Regulations. They support delegates on any kind of procedural matters happening at competitions and decide on unresolved and uncovered incidents. The WRC regularly publishes WCA internal reports to help all delegates and improve their knowledge of the Regulations. If you would like to take part in regulation development or propose changes you can reach out to the WRC by contacting them via mail or discussions on Github: https://github.com/thewca/wca-regulations."
     wrt:
-      info: "(competition results, result corrections)"
+      info: "The WRT is responsible for managing all data in the databases of the WCA, most importantly competition results and person data. They are posting results of WCA competitions to the WCA website, once the delegate of the competition has sent them in. Apart from posting competition results, the WCA Results Team is also responsible for making corrections to competition results if necessary and they are in charge of administering all WCA profile pages. This includes for example changing user data, like dates of birth, according to verified requests and managing profile picture requests."
     wst:
-      info: "(maintenance of software of the WCA)"
+      info: "The WST is responsible for the maintenance and management of all software of the WCA. They also develop new software for various tasks within the WCA whenever needed. If you have suggestions and ideas for further website development or if you would like to report errors and bugs, feel free to contact them! The WCA website is a huge open source project on Github, so if you are interested in helping to develop for the WCA, take a look here: https://github.com/thewca/worldcubeassociation.org."
```

Comment:
Nit: Github -> GitHub

Reply:
Also, this link won't actually show up as a link, it just shows up as plaintext. To make this clickable, you'll need to do a few things:

1. Change this to an actual `a` tag, like `<a href="...">text here...</a>`.
2. Change all the `info` keys to `info_html`.
2. Update [this line of code](https://github.com/jfly/worldcubeassociation.org/blob/2ec18dc0bca019a4daa479c14d4ca61648402c50/WcaOnRails/app/views/static_pages/contact.html.erb#L25) and [this line of code](https://github.com/jfly/worldcubeassociation.org/blob/2ec18dc0bca019a4daa479c14d4ca61648402c50/WcaOnRails/app/views/static_pages/contact.html.erb#L18) to use `info_html` instead of `info`.